docs/fr/HISTORIQUE.md

```diff
@@ -0,0 +1,111 @@
+# Historique des versions
+
+uPortal est un projet mature en continuel développement depuis 1999. Cette page fournie une information générale sur les précédentes versions de uPortal.
+
+## Tables de compatibilité
+
+### Java: Version(s) Supportées
+
+|     |  9  |  8  |  7  |  6  |  5  |
+| --- | --- | --- | --- | --- | --- |
+| 5.x |  ?  |  Y  |  N  |  N  |  N  |
+| 4.3 |  N  |  Y  |  N  |  N  |  N  |
+| 4.2 |  N  |  Y  |  Y  |  N  |  N  |
+| 4.1 |  N  |  N  |  Y  |  N  |  N  |
+| 4.0 |  N  |  N  |  Y  |  Y* |  N  |
+| 3.2 |  N  |  N  |  ?  |  Y  |  N  |
+| 3.1 |  N  |  N  |  ?  |  Y  |  N  |
+| 3.0 |  N  |  N  |  ?  |  Y  |  N  |
+
+* Requires 1.6.0_26 or later
+
+### Spécifications de Servlet Requises
+
+|     | 3.1 | 3.0 | 2.5 | 2.4 | Tomcat Version |
+| --- | --- | --- | --- | --- | -------------- |
+| 5.x |  Y  |  N  |  N  |  N  | 8 or 8.5       |
+| 4.3 |  Y  |  N  |  N  |  N  | 8              |
+| 4.2 |  ?  |  Y  |  N  |  N  | 7 (maybe 8)    |
+| 4.1 |  N  |  Y  |  N  |  N  | 7              |
+| 4.0 |  N  |  N  |  Y  |  N  | 6 or 7         |
+| 3.2 |  N  |  N  |  Y  |  N  |                |
+| 3.1 |  N  |  N  |  Y  |  N  |                |
+| 3.0 |  N  |  N  |  Y  |  Y  |                |
+
+### Spécifications de Portlet supportées
+
+|     | 2.0 (JSR-286) | 1.0 (JSR-168) |
+| --- | ------------- | ------------- |
+| 5.x |       Y       |       Y       |
+| 4.3 |       Y       |       Y       |
+| 4.2 |       Y       |       Y       |
+| 4.1 |       Y       |       Y       |
+| 4.0 |       Y       |       Y       |
+| 3.2 |       N       |       Y       |
+| 3.1 |       N       |       Y       |
+| 3.0 |       N       |       Y       |
+
+## Version de uPortal depuis la 4.0
+
+### 5.2
+
+  - [5.2.1 06-Aug-2018](https://github.com/Jasig/uPortal/releases/tag/v5.2.1)
```

Comment:
:question: "Aug' -> "août"?

Reply:
De Fait - correction en cours

Reply:
Done

Reply:
Sorry for multiple commits, it was my first time.